community/kernel/src/main/java/org/neo4j/kernel/impl/locking/LockClientStateHolder.java

```diff
@@ -23,7 +23,7 @@
 
 /**
  * State control class for Locks.Clients.
- * Client state represent current Locks.Client state: OPEN/CLOSED
+ * Client state represent current Locks.Client state: OPEN/STOPPED
```

Comment:
"Open" and "Stopped" are not duals, so the wording ends up weird. I'm think it's better to keep "Closed" and introduce "Terminated" as the end-state.